### ISSUE TITLE
Some exceptions/errors occurred after running the Quick Start Code

### ISSUE THREAD

**Issue by ChristianEssen** [MaartenGr/BERTopic#1009]
Hi Maarten,

Thank you very much for your great work!
Unfortunately I get some error messages after running the Quick Start Code:

Traceback (most recent call last):
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\transformers\modeling_utils.py", line 415, in load_state_dict
    return torch.load(checkpoint_file, map_location="cpu")
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\torch\serialization.py", line 777, in load
    with _open_zipfile_reader(opened_file) as opened_zipfile:
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\torch\serialization.py", line 282, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\transformers\modeling_utils.py", line 419, in load_state_dict
    if f.read(7) == "version":
  File "C:\ProgramData\Python39\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 1767: character maps to <undefined>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\ReviewMining\RevMioAna\revmioanascr\bertopic01.py", line 7, in <module>
    topics, probs = topic_model.fit_transform(docs)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\bertopic\_bertopic.py", line 342, in fit_transform
    self.embedding_model = select_backend(self.embedding_model,
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\bertopic\backend\_utils.py", line 80, in select_backend
    return SentenceTransformerBackend("all-MiniLM-L6-v2")
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\bertopic\backend\_sentencetransformers.py", line 43, in __init__
    self.embedding_model = SentenceTransformer(embedding_model)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\sentence_transformers\SentenceTransformer.py", line 95, in __init__
    modules = self._load_sbert_model(model_path)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\sentence_transformers\SentenceTransformer.py", line 840, in _load_sbert_model
    module = module_class.load(os.path.join(model_path, module_config['path']))
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\sentence_transformers\models\Transformer.py", line 137, in load
    return Transformer(model_name_or_path=input_path, **config)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\sentence_transformers\models\Transformer.py", line 29, in __init__
    self._load_model(model_name_or_path, config, cache_dir)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\sentence_transformers\models\Transformer.py", line 49, in _load_model
    self.auto_model = AutoModel.from_pretrained(model_name_or_path, config=config, cache_dir=cache_dir)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\transformers\models\auto\auto_factory.py", line 464, in from_pretrained
    return model_class.from_pretrained(
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\transformers\modeling_utils.py", line 2301, in from_pretrained
    state_dict = load_state_dict(resolved_archive_file)
  File "C:\ReviewMining\RevMioAna\revmioanaenv\lib\site-packages\transformers\modeling_utils.py", line 431, in load_state_dict
    raise OSError(
OSError: Unable to load weights from pytorch checkpoint file for 'C:\Users\Christian Knobloch/.cache\torch\sentence_transformers\sentence-transformers_all-MiniLM-L6-v2\pytorch_model.bin' at 'C:\Users\Christian Knobloch/.cache\torch\sentence_transformers\sentence-transformers_all-MiniLM-L6-v2\pytorch_model.bin'. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True. niLM-L6-v2\pytorch_model.bin' at 'C:\Users\Christian Knobloch/.cache\torch\sentence_transformers\sentence-transformers_all-MiniLM-L6-v2\pytorch_model.bin'. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True.

Any help will be highly appreciated!
Christian

**Comment by MaartenGr** [comment]
This seems like it is an issue with your environment. The only relevant issue I can find is [this one](https://github.com/UKPLab/sentence-transformers/issues/789) but it does not give much clarity. I would advise starting from a fresh environment, remove any cached SBERT models, and re-install BERTopic there. 

One other thing that might be relevant here is that your pytorch version might be too old for that model, so starting from a fresh environment here might help.

**Comment by ChristianEssen** [comment]
Hi Maarten,

Thank you for your super-quick response!
I used several new environments, none of them worked – it drove me crazy ;-).
Shortly after I posted the script I found a _.cache_-folder in my _User_-folder with _torch_- and _sentence_transformers_-folders and deleted it. (I think, that is what you meant by "remove any cached SBERT models".)
And now everything works fine! 

Thanks again for your great work and greetings
Christian
